test/fixtures/variant_lists.json

```diff
@@ -0,0 +1,225 @@
+{
+    "type": "Resource",
+    "body": [
+        {
+            "type": "Term",
+            "annotations": [],
+            "id": {
+                "type": "Identifier",
+                "name": "-variant-list-in-term"
+            },
+            "value": {
+                "type": "VariantList",
+                "variants": [
+                    {
+                        "type": "Variant",
+                        "key": {
+                            "type": "VariantName",
+                            "name": "key"
+                        },
+                        "value": {
+                            "type": "Pattern",
+                            "elements": [
+                                {
+                                    "type": "TextElement",
+                                    "value": "Value"
+                                }
+                            ]
+                        },
+                        "default": true
+                    }
+                ]
+            },
+            "attributes": [],
+            "comment": null
+        },
+        {
+            "type": "Term",
+            "annotations": [],
+            "id": {
+                "type": "Identifier",
+                "name": "-variant-list-in-term-attr"
```

Comment:
This `Term` parsed fine up to the broken `Attribute`. I filed #132 to discuss the desired behavior of the parser in such cases.